packages/autotask-client/src/api.ts

```diff
@@ -77,6 +77,24 @@ export class AutotaskClient extends BaseApiClient {
     return this.updateCode(autotaskId, encodedZippedCode);
   }
 
+  public async listAutotaskRuns(autotaskId: string): Promise<void> {
+    return this.apiCall(async (api) => {
+      return await api.get(`/autotasks/${autotaskId}/runs`);
+    });
+  }
+
+  public async getAutotaskRun(autotaskRunId: string): Promise<void> {
+    return this.apiCall(async (api) => {
+      return await api.get(`/autotasks/runs/${autotaskRunId}`);
```

Comment:
Part of me wants this endpoint to be: 

```
/autotasks/${autotaskId}/runs/${autotaskRunId}
```

Fits the existing schema better and (optimizing for an incredibly unlikely scenario 😉 ) if there was ever an `autotaskId` and/or `autotaskRunId`  of "runs", it's possible something could break. It would force the user to provide a second argument (`autotaskId`) to this function, so in that sense, there is a minor convenience with the current implementation. 

But I think they should have the value of `autotaskId` available based on the `list` response:

```
{
  items: [
    {
      autotaskRunId: 'c91b8072-d8d6-4ba9-9380-3e00034f889f',
      autotaskId: '09b25a34-6636-46b3-b099-ecb6a4b44d8b',
      trigger: 'manual-api',
      status: 'success',
      createdAt: '2022-04-20T15:35:28.106Z'
     },
     {
      autotaskRunId: '98903fd9-99fb-4ae8-871a-0332a0e2d85e',
      autotaskId: '09b25a34-6636-46b3-b099-ecb6a4b44d8b',
      trigger: 'manual-api',
      status: 'success',
      createdAt: '2022-04-18T17:22:16.474Z'
     },
     ...
  ],
  next: '{"tenantAutotaskId":"DEFAULT_TENANT|09b25a34-6636-46b3-b099-ecb6a4b44d8b","createdAt":"2022-04-04T14:01:08.475Z","tenantId":"DEFAULT_TENANT","autotaskRunId":"797cbc8e-457f-41a2-b7b8-462d1b9a6e88"}'
}
```

Any thoughts @pasevin (or @spalladino / @zeljkoX )?

Reply:
> if there was ever an autotaskId and/or autotaskRunId of "runs", it's possible something could break

Not sure I follow this, could you elaborate?

I agree that users can have the autotask id along with the run id, but since we don't really need the autotask id for the lookup on the backend, I'd keep the API simple and **not** require it, leaving @pasevin's implementation as-is. But my vote may change depending on what you bring up regarding what could break!

Reply:
Oh, this is a silly breakage and optimizing for it is somewhat tongue in cheek, but imagine we changed the way these ids are generated and generated a `autotaskId` of "runs". 

An incoming `listAutotaskRuns` request (`GET .../autotasks/runs/runs`) would presumably end up hitting the `getAutotaskRun` endpoint. I think the bigger issue was more around the consistency of the structure of these endpoints, but totally get the preference for keeping API simple.

Reply:
Got it! My vote then is on keeping it simple. @pasevin, you're the tie-breaker and get to choose ;-)

Reply:
From what I understand how the data is structured and how autotask runs relate to autotasks I have to agree with @mok0230. My OCD wants me to follow the RESTful best practices and use nesting to better illustrate the relations and hierarchy (`autotasks/{$auotaskId}/runs/${autotaskRunId}).